.github/workflows/release.yml

```diff
@@ -81,12 +90,13 @@ jobs:
         run: git push
 
       - name: 🚀 Release ${{ env.VERSION }}
-        uses: softprops/action-gh-release@e7a8f85e1c67a31e6ed99a94b41bd0b71bbee6b8 # v2.0.9
+        uses: softprops/action-gh-release@c062e08bd532815e2082a85e87e3ef29c3e6d191 # v2.0.8
         with:
           files: ${{ env.NAME }}-${{ env.VERSION }}.vsix
           tag_name: v${{ env.VERSION }}
+          body: ${{ env.RELEASE_NOTES }}
```

Comment:
Somehow this didn't work 🧐

![image](https://github.com/user-attachments/assets/ce2b5e25-a634-4b35-8423-124f76ba0e72)

The GitHub Release Notes are not properly shown anymore.